app/views/layouts/application.html.haml

```diff
@@ -32,5 +32,5 @@
       = yield
     #error_tips
     %footer
-      Review it! #{version} © 2014-2015
+      Review it! #{version} © #{copyright_year(2014)}
```

Comment:
Initial year will never change, hehehe, so I think just use #{Time.now.year} here is enough.

Reply:
Fixed